src/main/java/controller/Controller.java

```diff
@@ -21,7 +21,9 @@ public Rubric addRubric(Rubric rubric) throws Exception {
     }
 
 
-    public void addCriteriaToRubric(Rubric rubric, Criteria criteria) throws Exception {
+    public void addCriteriaToRubric(String rubricName, Criteria criteria) throws Exception {
```

Comment:
You can now pass the rubric name instead of the rubric itself and the getRubric method will handle returning the right rubric based on the passed name.